maps/CopycatsHouse2F.asm

```diff
@@ -70,7 +70,7 @@ Copycat:
 .Default_Merge_3a:
 	faceplayer
 	variablesprite SPRITE_COPYCAT, SPRITE_LASS
-	special MapCallbackSprites_LoadUsedSpritesGFX
+	special Special_MapCallbackSprites_LoadUsedSpritesGFX
```

Comment:
Can this have a shorter name, like `Special_LoadUsedSpritesGFX`?

Reply:
Sure thing, will do it tomorrow along with anything else that might come up. Assuming #456 gets merged first, I'll also try to do the merge myself

Reply:
91f97f08